### ISSUE TITLE
Embed for fastcompany.com not generated correctly

### ISSUE THREAD

**Issue by mackuba** [bluesky-social/atproto#1986]
**Describe the bug**

For this link: https://www.fastcompany.com/91002426/everyone-wants-to-leave-twitter-why-arent-they-on-bluesky

Embed on Bluesky:

<p><img width="580" alt="Screen Shot 2023-12-22 at 19 31 48" src="https://github.com/bluesky-social/atproto/assets/28465/e7154529-d571-40d0-9fa2-2b149d9e5550"></p>

Embed on Mastodon:

<p><img width="575" alt="Screen Shot 2023-12-22 at 19 31 54" src="https://github.com/bluesky-social/atproto/assets/28465/667a1cdf-3b57-4e57-a33d-225775304b0c"></p>

**Details**

- Operating system: macOS 12.7.1
- Browser: Safari 17.2.1

**Comment by bnewbold** [comment]
Yeah, like many websites they seem to be hostile to non-browser connections. Simple `User-Agent` spoofing doesn't seem to work.

The ideal approach would be to pressure sites like this to just be normal and allow access to at least minimal metadata (seems like it would be in their own interests!). Pragmatically we may eventually set up headless browsers to bypass some forms of bot detection, or pay a vendor (ugh).
